<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-11-05T10:05:49 IST" name="Suite" finished-at="2024-11-05T10:05:58 IST" duration-ms="9121">
    <groups>
    </groups>
    <test started-at="2024-11-05T10:05:49 IST" name="Test" finished-at="2024-11-05T10:05:58 IST" duration-ms="9121">
      <class name="com.cb.tests.OrangeHRMTests">
        <test-method is-config="true" signature="setUp([Ljava.lang.Object;)[pri:0, instance:com.cb.tests.OrangeHRMTests@5b1669c0]" started-at="2024-11-05T10:05:51 IST" name="setUp" finished-at="2024-11-05T10:05:58 IST" duration-ms="7235" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.Object;@1e6308a9]]>
              </value>
            </param>
          </params>
          <exception class="com.cb.exceptions.PropertyFileValueException">
            <message>
              <![CDATA[Property name URL not found.please check config.properties]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.cb.exceptions.PropertyFileValueException: Property name URL not found.please check config.properties
at com.cb.utils.ReadPropertyFile.get(ReadPropertyFile.java:39)
at com.cb.driver.Driver.initDriver(Driver.java:49)
at com.cb.tests.BaseTest.setUp(BaseTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- com.cb.exceptions.PropertyFileValueException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginLogOutTest(java.util.Map)[pri:0, instance:com.cb.tests.OrangeHRMTests@5b1669c0]" started-at="2024-11-05T10:05:58 IST" name="loginLogOutTest" description="Verify user is able to login successfully or not" data-provider="getData" finished-at="2024-11-05T10:05:58 IST" duration-ms="0" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{TestName=loginLogOutTest, UserName=Admin, Execute=yes, Browser=chrome, Password=YWRtaW4xMjM=}]]>
              </value>
            </param>
          </params>
          <exception class="com.cb.exceptions.PropertyFileValueException">
            <message>
              <![CDATA[Property name URL not found.please check config.properties]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.cb.exceptions.PropertyFileValueException: Property name URL not found.please check config.properties
at com.cb.utils.ReadPropertyFile.get(ReadPropertyFile.java:39)
at com.cb.driver.Driver.initDriver(Driver.java:49)
at com.cb.tests.BaseTest.setUp(BaseTest.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- com.cb.exceptions.PropertyFileValueException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginLogOutTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.cb.tests.OrangeHRMTests@5b1669c0]" started-at="2024-11-05T10:05:58 IST" name="tearDown" finished-at="2024-11-05T10:05:58 IST" duration-ms="238" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.cb.tests.OrangeHRMTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
